config.yaml.defaults

```diff
@@ -228,6 +228,10 @@ services:
 misc:
   days_to_keep_unsaved_candidates: 7
   public_group_name: "Sitewide Group"
+  # Use a named cosmology from `astropy.cosmology.parameters.available`
+  # or supply the arguments for an FLRW cosmological instance
```

Comment:
I think this comment could be more descriptive in terms of specifying a custom FLRW cosmological instance, i.e. explaining that the `FlatLambdaCDM` and `LambdaCDM` are the two FLRW classes made available based on having a `"flat"` flag set. It seems like from the astropy documentation that there are other classes inheriting from FLRW available that aren't in the utils/cosmology.py (like `FlatwCDM`) so just saying "arguments for an FLRW cosmological instance" may be misleading. Of course, I don't know how relevant those unused cosmologies actually are from practical astronomy sense so this may be a moot point.

Reply:
Done.